docs/sdk/server-side-sdks/dotnet/dotnet-usage.md

```diff
@@ -124,3 +124,50 @@ To disable realtime updates, pass in the `disableRealtimeUpdates` option to the
         .SetLogger(LoggerFactory.Create(builder => builder.AddConsole()))
         .Build();
 ```
+
+## Evaluation Hooks
+
+Using evaluation hooks, you can hook into the lifecycle of a variable evaluation to execute code before and after execution of the evaluation.
+
+Note: The evaluation will wait for all hooks before returning the variable evaluation.
```

Comment:
```suggestion
Note: The evaluation will wait for all hooks before returning the variable evaluation. This will also block future variable evaluations until it returns.
```

Reply:
@jsalaber i think this is also a good thing to point out